dcm4che-imageio/src/main/java/org/dcm4che3/imageio/plugins/dcm/DicomImageReader.java

```diff
@@ -429,8 +454,14 @@ public BufferedImage read(int frameIndex, ImageReadParam param)
         return ovlyData;
     }
 
-    @SuppressWarnings("resource")
-    private ImageInputStream iisOfFrame(int frameIndex) throws IOException {
+    /** Generate an image input stream for the given frame, -1 for all frames (video, multi-component single frame)
+     * Does not necessarily support the length operation without seeking/reading to the end of the input.
+     * 
+     * @param frameIndex
+     * @return
+     * @throws IOException
+     */
+    public ImageInputStream iisOfFrame(int frameIndex) throws IOException {
```

Comment:
Making this public allows reading the raw bytes from the input, which can be useful if the actual image encoding isn't being changed, but a file is just being written from the original data.